public/index.html

```diff
@@ -35,7 +35,28 @@
 
   <body class="flex flex-col h-full">
     <noscript>
-      <h3>You need to enable JavaScript to run this app.</h3>
+      <link href="noscript.css" rel="stylesheet">
```

Comment:
in this case we will always be loading noscript.css anyway. at this point we should have full access to tailwind, so why not use it

Reply:
I don't think we do. When I access the page with javascript disabled tailwind classes don't seem to work. According to Firefox Tailwind classes seem to be inlined somewhere, which would explain this behavior. With `noscript.css` being loaded under the noscript tag, it will only be downloaded and executed in that block.

![image](https://user-images.githubusercontent.com/16887983/76548718-f43be400-646d-11ea-9cd3-d191559f7610.png)

Reply:
I see @RodrigoLeiteF!